lib/cdo/infra_production_topic.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../../deployment'
+require 'cdo/slack'
+require 'tzinfo'
+
+module InfraProductionTopic
+  def self.set_dtp_commit(commit)
+    msg = "DTP Status: #{commit} @ #{current_time_pacific}"
+    Slack.update_topic 'infra-production', msg
+  end
+
+  private_class_method def self.current_time_pacific
```

Comment:
TODO(asher): Refactor this method and the corresponding method in `lib/cdo/infra_test_topic.rb` to a common location.